### PR TITLE
Transtale upload tokens to french

### PR DESCRIPTION
**Pull request by hutnikau**

### CODE REVIEW

**Comment by krampstudio** [comment]
Sorry could you please reassign it, I won't be able to treat it today.

File: locales/fr-FR/messages.po [hunk 1]
```diff
@@ -25,6 +25,9 @@ msgstr "Performance moyenne (risque de lenteurs)"
 msgid "Average rendering time"
 msgstr "Temps de rendu moyen"
 
+msgid "Average upload speed"
+msgstr "Bande Passante Moyenne"
```
Comment (jsconan) [inline locales/fr-FR/messages.po]: The JIRA ticket mention another sentence here. It should be:

```
msgid "Average upload speed"
msgstr "Bande Passante Montante Moy."
```

File: locales/fr-FR/messages.po [hunk 2]
```diff
@@ -25,6 +25,9 @@ msgstr "Performance moyenne (risque de lenteurs)"
 msgid "Average rendering time"
 msgstr "Temps de rendu moyen"
 
+msgid "Average upload speed"
+msgstr "Bande Passante Montante Moy"
```
Comment (jsconan) [inline locales/fr-FR/messages.po]: I did not pay attention, but the AC also mention a dot at the end of the sentence:
`"Bande Passante Montante Moy."`

File: locales/fr-FR/messages.po [hunk 3]
```diff
@@ -76,6 +85,12 @@ msgstr ""
 msgid "Low bandwidth"
 msgstr "Bande passante faible"
 
+msgid "Low upload speed"
+msgstr "Bande Passante non conforme"
+
+msgid "Max upload speed"
+msgstr "Bande Passante Montante Max"
```
Comment (jsconan) [inline locales/fr-FR/messages.po]: I did not pay attention, but the AC also mention a dot at the end of the sentence:
`"Bande Passante Montante Max."`

**Pull request merged**